cluster/definition.yaml

```diff
@@ -82,15 +82,15 @@ metadata:
           validation:
           - required: false
 spec:
+  group: aws.platformref.crossplane.io
```

Comment:
have you tested these changes in the UbC console also? I suspect the UI metadata (e.g. line 59 above) would also need to be updated to work OK

Reply:
@jbw976 I tested according to the README with a hosted cluster. What do I need to do to test in UBC?

Reply:
I did test it in UBC and made some changes. This should be ready for review.